### ISSUE TITLE
Missing aria-labels in text editor toolbar

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#3818]
Three buttons on the Text Editor Toolbar (`close tags`, `fn` and `Distraction free writing mode`) do not have aria-labels (all other buttons do). `close tags` and `Distraction free writing mode` have a title instead. `fn` does have neither a title nor an aria-label. 

`close tags` should have an `aria-label="Close tags"` (or Close all open tags) and `fn` should have an `aria-label="Footnote"`.

Distraction free writing mode does have an aria-label in the Visual editor (`Distraction-free writing mode (^⌥W)`).

![image](https://github.com/user-attachments/assets/55f59d94-77a5-408b-8922-f4bc8ebd27c0)

**Comment by cmurtagh** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pressbooks-Product-Backlog-60ae7d807f6e370010b3d5be/issues/pressbooks/pressbooks/3818?planning-poker) with Zenhub @arzola @fdalcin @MichelleWeremczuk @richard015ar @tw77

**Comment by SteelWagstaff** [comment]
@cmurtagh @arzola I've added a PR to address this: #4233

**Comment by tw77** [comment]
aria-labels added as needed on dev (state change / changing aria-label in place for footnote button too), ready for release
